operator/gitops/argocd/tekton-results/configmap.yaml

```diff
@@ -10,3 +10,32 @@ metadata:
     app.kubernetes.io/version: v0.4.0
   name: tekton-results-database
   namespace: tekton-results
+---
+kind: ConfigMap
```

Comment:
Can you confirm that this configmap contains default settings that are unlikely to be modified by operators?

Reply:
That's correct, there are no operators that currently change anything in the configmap.